README.md

```diff
@@ -196,7 +196,9 @@ The official OCS samples are divided in multiple categories depending on the sce
   </tr>
   <tr>
     <th align="middle" colspan="2">
-      <h2>Types, Streams, and Retrieving Data</h2>
+      <h2>        
+      <a href="https://github.com/osisoft/OSI-Samples-OCS/docs/COMMON_ACTIONS.md">
```

Comment:
the existing ones are all named <CATEGORY>_README.md, but none of these files say 'readme' at the end of the file name. For consistency, should we add _README to these files name? I don't personally like it at the end of the file name, so I'd be in favor of removing it from the existing files as well :)

Reply:
I think this is a fair request